acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -135,6 +135,13 @@ def report_query(node)
     assert_allowed
   end
 
+  step 'static file content endpoint' do
+    # We'd actually need to perform a commit and use its code-id in order to
+    # get back a 200, but we know that a 400 means we got past authorization
```

Comment:
Would it be worth, at least, adding a test case which makes a request without a client cert and verifies that the response is a 404?

This is rather weak test, but ... aside from adding the other case I just mentioned, I don't have any other ideas on how to immediately improve this.  I am assume other testing of `/static_file_content` endpoint is coming in the near future from @erikPrime.